modules/sff/module/auto/sff.yml

```diff
@@ -148,6 +148,12 @@ sff_module_types: &sff_module_types
     desc: "100BASE-LX"
 - 100_BASE_FX:
     desc: "100BASE-FX"
+- 100G_BASE_LR:
```

Comment:
You should group the 100G enums together.

Reply:
grouped 100G...